source/guide_guillotine.rst

```diff
@@ -0,0 +1,280 @@
+.. highlight:: console
+
+.. author:: Jan Philip Bernius <https://janphilip.bernius.net>
+
+.. tag:: lang-ruby
+
+##########
+Guillotine
+##########
+
+.. tag_list::
+
+Guillotine_ is a simple URL Shortener written in Ruby and powers GitHubs URL Shortener Git.io_.
+It supports multiple storage adapters, including :manual:`MySQL <database-mysql>`, :lab:`PostgreSQL <guide_postgresql>`, SQLite, :lab:`Redis <guide_redis>`, Riak and Cassandra.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Ruby <lang-ruby>` and its package manager :manual_anchor:`gem <lang-ruby.html#gem>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+Guillotine is released under the `MIT License`_.
+
+Prerequisites
+=============
+
+We're using :manual:`Ruby <lang-ruby>` in the stable version 10:
```

Comment:
Do we need 10 or 2.6?

Reply:
Uups, missed that one when Copy/Pasting 🙃